src/WROVER_KIT_LCD.h

```diff
@@ -149,6 +149,7 @@ class WROVER_KIT_LCD : public Adafruit_GFX {
         void      setRotation(uint8_t r);
         void      invertDisplay(boolean i);
         void      scrollTo(uint16_t y);
+        void      setupScrollArea(uint16_t TFA, uint16_t BFA);
```

Comment:
what is TFA and BFA? Please do not use capitals for local variables :)

Reply:
So what does this do? Any example?

Reply:
TFA = Top Fixed Area, BFA = Bottom Fixed Area.

I'll lowercase it as on [Bodmer's library](https://github.com/Bodmer/TFT_ILI9341/blob/master/examples/TFT_Terminal/TFT_Terminal.ino#L138) and update the PR right away.

The example is in the link of the first comment, I've made more of those since

- https://www.youtube.com/watch?v=434LDAfpGjE
- https://www.youtube.com/watch?v=2AGu0f8MKBk
- https://www.youtube.com/watch?v=_GrhF4i2eaY

Reply:
More insight on what it does: the `scrollTo` existing function scrolls the whole screen.
By using `setupScrollArea` before that, it is possible to scroll only a portion of the screen with tfa as top offset and bfa as bottom offset.

Reply:
Example should be added in an Arduino `examples` folder :)

Reply:
Now there's a scrollable WiFi scan in the examples folder :D